test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -1243,6 +1243,31 @@ spec setup =
             t = table_builder [["x", ["1", "2", "3"]]]
             t.at "x" . const (t.at "x") . should_fail_with Illegal_Argument
 
+    Test.group prefix+"Table.make_constant_column" <|
+        Test.specify "Should allow the creation of constant columns" <|
+            t = table_builder [["x", ["1", "2", "3"]]]
+            t.make_constant_column True . to_vector . should_equal [True, True, True]
+            t.make_constant_column 12 . to_vector . should_equal [12, 12, 12]
+            t.make_constant_column 12.3 . to_vector . should_equal [12.3, 12.3, 12.3]
+            t.make_constant_column "asdf" . to_vector . should_equal ["asdf", "asdf", "asdf"]
+
+        Test.specify "Should allow the creation of constant columns on a table with no rows" <|
+            t = table_builder [["x", ["1", "2", "3"]]]
+            empty = t.take 0
+            constant = empty.make_constant_column 42
+            empty2 = empty.set constant
+            empty2.column_names.should_equal ['x', '42']
+
+        Test.specify "Should create a column of the correct type on a table with no rows" <|
+            t = table_builder [["x", ["1", "2", "3"]]]
+            empty = t.take 0
+            [[False, .is_boolean], [42,  .is_integer], ["42",  .is_text], ["foo", .is_text], [1.1, .is_floating_point]].map pair->
+                value = pair.at 0
+                pred = pair.at 1
+                c = empty.make_constant_column value
+                pred c.value_type . should_be_true
+                pred ((empty.set c).at c.name . value_type) . should_be_true
```

Comment:
Since you have added the special case for `Nothing`, let's add a test checking the `Nothing` as well. At least for in-memory, because I'm not sure what will happen in DB.

Reply:
Done